lib/msf/core/modules/external/bridge.rb

```diff
@@ -238,12 +238,33 @@ def handle_exception(error)
   end
 end
 
+class Msf::Modules::External::JavaBridge < Msf::Modules::External::Bridge
+  def self.applies?(module_name)
+    module_name.match? /\.jar$/
```

Comment:
When regex is not required, `end_with` is preferred for matching the end of strings.

```suggestion
    module_name.end_with?('.jar')
```